src/select/__tests__/select.test.tsx

```diff
@@ -439,103 +413,6 @@ describe.each([false, true])('expandToViewport=%s', expandToViewport => {
     });
   });
 
-  describe('Disabled state', () => {
```

Comment:
Moved this section to a new file and added a new test to it ("hides disabled reason when the option is scrolled away")